.gitattributes

```diff
@@ -0,0 +1,13 @@
+/bin export-ignore
+/node_modules export-ignore
+/tests export-ignore
+.editorconfig export-ignore
+.gitattributes export-ignore
+.gitignore export-ignore
+.travis.yml export-ignore
+CHANGELOG.md export-ignore
```

Comment:
I am not sure about this one. Shouldn't we leave this file in an export?

Reply:
It's not the export to the WordPress.org repo, but you are right, we might want to keep it.